keps/sig-node/1769-memory-manager/README.md

```diff
@@ -577,96 +595,99 @@ the health of the service?**
   
   *For a pod author:*
   
-  * Pod succeeded to start. You have two options to verify if containers pinned to the NUMA node
+  * Pod succeeded to start. You have two options to verify if containers are pinned to the NUMA node
     - Via pod resources API, you will need to connect to grpc socket and get information from it, see [pod resource API doc page](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources) for more information. 
     - Checking the relevant container CGroup under the node.
 
   * Pod failed to start because of the admission error.
     
-    To understand the reason you will need to check via pod resources API 
-    the amount of allocatable memory and memory reserved by containers.
+    To understand the reason, you will need to check via pod resources API the amount of allocatable memory and memory reserved by containers.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  This does not seem relevant to this feature.
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
-  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-  implementation difficulties, etc.).
-  Currently, for the pod author, it is impossible to know containers NUMA pinning without access to the node.
+This does not seem relevant to this feature.
```

Comment:
What percent of errors in the metrics (memory_manager_pinning_requests_total`, `memory_manager_pinning_errors_total`) are expected?  If it's a low number, lets say that.

Reply:
I think memory_manager_pinning_error_total should be zero on healthy node with enough resources.

memory_manager_pinning_requests_total however, depends on the number of guaranteed workloads running on the node/cluster

Reply:
So the question is relevant.  Let's get an answer filled in.

Reply:
Sure thing, amending.